original_post_discovery.py

```diff
@@ -117,10 +117,10 @@ def resolve(urls):
     # the best chance of finding a match. Some silos allow several
     # different permalink formats to point to the same place (e.g.,
     # facebook user id instead of user name)
-    syndication_url = source.canonicalize_syndication_url(
-      util.follow_redirects(syndication_url).url)
-    originals.update(_posse_post_discovery(
-      source, activity, syndication_url, fetch_hfeed))
+    syndication_url = source.canonicalize_url(syndication_url)
+    if syndication_url:
+      originals.update(_posse_post_discovery(
+        source, activity, syndication_url, fetch_hfeed))
```

Comment:
oh interesting, so I was imagining it'd still follow redirects if the URL is unrecognized. This is more like: don't follow redirects (unless a service's implementation chooses to), we can always construct a canonical url out of the given url.

I'm fine with that change, just want to make sure we're on the same page.